### DISCUSSION TITLE
screen mirroring, hdmi

### DISCUSSION THREAD

**Discussion by nik8765** [SafeExamBrowser/seb-win-refactoring#320] [Q&A]
i just became an assistant this year and dont know how this works. does seb detect if students are screen mirroring or using hdmi cabel to connect to other devices while taking an exam. specificaly our university is using exam. net. i found that for earlier versions it wasnt possible yet for windows. does the 3.3.1 version detect this kind of cheating or not yet?

**Comment by dbuechel** [comment]
Yes, as of SEB 3.3.1 one can restrict the amount and type of displays (internal or external) allowed for a session. See https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane for the exact details.
